core/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -67,4 +67,18 @@ package object fixedpoint {
         case (_, _)                 => NilF()
       }
   }
+
+  /** Encodes a function that may diverge.
+    */
+  type Partial[A] = Nu[A \/ ?]
+  type ⊥[A] = Partial[A]
```

Comment:
Yeah, got that. It's mainly `now(a: A): ⊥[A]` that seems perverse to me. I would expect something called `⊥` to always diverge, but maybe I'm just missing the reference.